server/config/disaster_dashboard/Earth.textproto

```diff
@@ -394,3 +425,99 @@ categories {
     }
   }
 }
+
+categories {
+  title: "Extreme Heat"
+  blocks {
+    columns {
+      tiles {
+        type: DISASTER_EVENT_MAP
+        disaster_event_map_tile_spec: {
+          event_type_keys: "heat"
+        }
+      }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Number of extreme heat events last year"
+      #   stat_var_key: "heat_count"
+      # }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Total area affected by extreme heat events last year"
+      #   stat_var_key: "heat_area"
+      # }
+    }
+    columns {
+      tiles {
+        type: TOP_EVENT
+        title: "Most severe heat differences in the last year"
```

Comment:
nit: suggestion for title string:

"Most severe heat events in the last year" OR "Greatest differences in maximum temperatures compared to historical monthly averages in the last year"

Reply:
thanks, used the shorter title, and used the longer title as description text.